src/components/transport_manager/test/transport_manager_default_test.cc

```diff
@@ -145,18 +160,17 @@ TEST(TestTransportManagerDefault, Init_LastStateUsed) {
   EXPECT_CALL(transport_manager_settings,
               transport_manager_tcp_adapter_network_interface())
       .WillRepeatedly(ReturnRef(network_interface));
-  EXPECT_CALL(transport_manager_settings, bluetooth_uuid())
-      .WillRepeatedly(Return(kBTUUID.data()));
```

Comment:
Should not be removed

Reply:
@jacobkeeler Reverted in 90937393b007f9641c8e5ef9953d102e371ab773